app/frontend/terrier/theme.ts

```diff
@@ -101,6 +102,9 @@ export default class Theme {
                 if (action.icon?.length) {
                     this.renderIcon(a, action.icon, iconColor)
                 }
+                if (action.img?.length) {
+                    a.img('.image-thumb', {src:  action.img})
```

Comment:
Is `.image-thumb` a css class that's defined in terrier-engine?

I think if possible I would prefer to add basic styling for this to each place where actions might be used (i.e. in tt-dropdowns.scss as well as in tt-page.scss, tt-modal.scss, and tt-pane.scss), which can then be overwritten in clypboard/hub/lycensed as necessary for those specific apps.

Reply:
.image-thumb doesnt have any styling, I added styling to the tt-dropdowns tho just to cap the width in the drop down